pa_matchbatch.sh

```diff
@@ -203,7 +220,7 @@ for currentItem in ${repoRevMap}; do
 	#    [[ "${ignored_repos}" =~ ${foss_name} ]]
 	# will not work when $ignored_repos contains e.g. puppet-agent
 	# and $foss_name is puppet.
-	if containsElement "${ignored_repos}" "${foss_name}"; then
+	if containsElement "${ignored_repos} .git" "${foss_name}"; then
```

Comment:
Why is there `.git` here?

Reply:
The .git directory was being included, and throwing errors, for me.

Reply:
I'm not seeing the same errors now, so going to remove this.